package.json

```diff
@@ -16,6 +16,7 @@
     "node": ">= 16"
   },
   "main": "./index.js",
+  "types": "./types/index.d.ts",
```

Comment:
I'm so sorry to spot it that late. But in order for this file to be published I would have to ask you to add `"types"` folder to the `"files"` array below, right after the `"lib"`.

Reply:
Don't worry, it's a good learning experience, I added it